services/registration/src/routes/application.ts

```diff
@@ -586,6 +586,11 @@ applicationRouter.route("/:id/actions/update-status").post(
         newStatus === StatusType.NOT_ATTENDING
       ) {
         // pass
+      } else if (
+        existingApplication.status === StatusType.CONFIRMED &&
```

Comment:
we shouldn't allow hackathon attendees to check themselves in. only hexlabs members should be able to change their check in status